### ISSUE TITLE
about voice embedding

### ISSUE THREAD

**Issue by audio-visual** [cmu-mlsp/reconstructing_faces_from_voices#9]
hi, 
I am very interested in your work, but it is not clear how to get the voice embedding, and the paper did not explain it in detail. 
May I ask, did you obtain this network by mapping voices to identities through a classification network? Or did you use any other fancy loss / architecture? 
Also, did you also only use the dataset mentioned in the paper for training? Or are additional datasets used?
Thanks!

**Comment by ydwen** [comment]
Hi Wuyang,

We did obtain the voice embedding network by training a speaker recognition model and taking the backbone (i.e. feature extractor). If I remember correctly, we just use the classification loss (i.e. cross-entropy loss). We tried different fancy loss functions but the performance is very similar.

We only used voxceleb1 in this work. We did try voxceleb2 in our following research and achieved better performance. You can consider either or both of them.

**Comment by audio-visual** [comment]
Thanks so much for your reply!

Yandong Wen ***@***.***> 于2023年3月13日周一 07:24写道：
